packages/style/scss/components/list-box.scss

```diff
@@ -30,18 +32,22 @@
     --padding: 8px 16px;
 
     gap: 9px;
+    min-width: 0;
+
     height: var(--height);
     padding: var(--padding);
-    overflow: hidden;
     color: var(--color);
-    white-space: nowrap;
-    text-overflow: ellipsis;
     list-style: none;
     background-color: var(--background-color);
     outline: none;
-
     cursor: var(--cursor);
 
+    > .truncante-item-box-content {
```

Comment:
```suggestion
    > .truncate-item-box-content {
```

Reply:
Also why not use the existing `truncate` class?

https://github.com/coveo/plasma/blob/master/packages/style/scss/typography/utility.scss#L66-L72

Reply:
Oh! I should go in the Utility more often 🙈